css/style.css

```diff
@@ -0,0 +1,1109 @@
+  border-radius: 5%;
+}
+
+.lat-post5 p {
+  font-size: small;
+  margin-left: 120px;
+}
+
+.latest-posts2,
+.latest-posts3 {
+  display: flex;
+  margin-top: 5px;
+  height: 650px;
+}
+
+.latest-posts4,
+.community-footer,
+.more-guides p {
+  width: 100%;
+  margin-top: 20px;
+  height: 50px;
+}
+
+.lat-img-container {
+  margin-top: 20px;
+  width: 200px;
+  height: 200px;
+  border-radius: 5%;
+  background-color: white;
+  margin-left: 38.3%;
+  margin-right: 38.3%;
+}
+
+.lat-post6 p,
+.lat-post6 header,
+.lat-post11 p,
+.lat-post11 header {
+  margin-left: 40px;
+  margin-right: 40px;
+  width: 700px;
+}
+
+h2 {
+  margin: 10px 40px 10px 40px;
+}
+
+.article-title h2 {
+  padding: 10px 10px 0 50px;
+  font-size: xx-large;
+  font-weight: 900;
+  font-family: "Cairo", sans-serif;
+}
+
+.lat-post6 button,
+.lat-post11 button {
+  width: 300px;
+  height: 60px;
+  margin-bottom: 20px;
+  font-size: x-large;
+  border-radius: 3%;
+  cursor: pointer;
+  font-family: "Nunito", sans-serif;
+}
+
+.lat-post6 header,
+.lat-post11 header {
+  font-size: xx-large;
+}
+
+.lat-paragraph {
+  font-size: x-large;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.latest-posts4,
+.community-footer {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.latest-posts4 > p,
+.community-footer > p {
+  width: 16%;
+  padding-bottom: 20px;
+  border-bottom: 5px solid black;
+}
+
+.latest-arrow {
+  background: inherit;
+}
+
+.clicks {
+  text-align: center;
+  margin-top: 40px;
+  background-color: #b5b6b7;
+  padding: 10px;
+}
+
+.click-date {
+  margin-top: 50px;
+  margin-bottom: 50px;
+  font-family: "Libre Baskerville", serif;
+  color: white;
+}
+
+.clicks header {
+  font-size: 40px;
+  margin-left: 20%;
+  margin-right: 20%;
+  font-family: "Cairo", sans-serif;
+}
+
+.clicks p {
+  font-size: x-large;
+  margin-left: 20%;
+  margin-right: 20%;
+  color: white;
+}
+
+.clicks button,
+.main-sect button {
+  height: 60px;
+  font-size: xx-large;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  cursor: pointer;
+}
+
+.click-img {
+  height: 400px;
+  width: 600px;
+  background-color: white;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 30px;
+}
+
+.community {
+  margin-top: 60px;
+}
+
+.community-top {
+  display: flex;
+  flex-direction: row;
+}
+
+.community-links,
+.guide-text {
+  writing-mode: vertical-rl;
+  text-orientation: upright;
+}
+
+.community-big {
+  margin-left: 40px;
+  width: 80%;
+  margin-bottom: 50px;
+}
+
+.community-big p {
+  font-family: "Nunito", sans-serif;
+}
+
+.sm-container {
+  float: left;
+  margin-left: 30px;
+  margin-right: 30px;
+  margin-bottom: 180px;
+}
+
+.small-container {
+  height: 115.85px;
+  width: 115.85px;
+  background-color: #cecece;
+  transform: rotate(-10deg);
+  border-radius: 5%;
+}
+
+.small-link {
+  text-decoration: none;
+  margin-bottom: 5px;
+  color: black;
+  border-bottom: 1px solid black;
+}
+
+.small-link:active {
+  color: black;
+}
```

Comment:
This selector and selectors on lines 388, 405, and 493 have the same style definition. Group selectors or create a new class.